src/main/webapp/sass/_core.scss

```diff
@@ -531,10 +531,9 @@ end sticky footer
 }
 
 .light-border {
-  border: solid #ccc;
-  border-width: 0.1em;
-  border-radius: 0.4em;
-  padding: 2px 4px 2px 3px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  padding: 16px;
 }
```

Comment:
.light-border and .highlight-box were both used on the settings page, so change the styles with two goals
- light-border and highlight-box should match visually
- Change both styles so contents (e.g. title font) match the contents of the new 2FA panel. There is a risk that these could drift, but my hope is SageAccountWeb is production-ready before drift happens.